spec/models/assignment_spec.rb

```diff
@@ -46,5 +46,21 @@
         @assignment.set_deadline_job
       }.to have_enqueued_job(AssignmentDeadlineSubmissionJob).on_queue('default')
     end
+
+    describe "#project_order" do
+      before do
+        @projects = []
+
+        (1..3).each do
+          @projects << FactoryBot.create(:project, assignment: @assignment,
+            author: FactoryBot.create(:user))
+        end
+      end
+
+      it "gives sorted assignment projects" do
+        expect(@assignment.project_order.map { |p| p.author.name })
+        .to eq(@projects.map { |p| p.author.name }.sort)
```

Comment:
🤔 testing same thing ?

Reply:
`@assignment.project_order.map { |p| p.author.name }` - this will be a list of author names of the projects `@assignment.project_order`
`@projects.map { |p| p.author.name }.sort` - this will be a *sorted* list of author names from the projects associated with the assignment. So this checks if the projects from `@assignment.project_order`are sorted by author name 😅